jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -368,6 +368,8 @@ def _change_overlay(self, *args):
         self._preset_args_changed()
 
     def _mark_visible(self, viewer_id, overlay=None):
+        if overlay == 'default' and overlay not in self._overlays:
+            return

```

Comment:
Can we generalize this even further?  And should we return False since this is expecting a boolean?

```suggestion
        if overlay not in self._overlays:
            return False

```